docs/common/istio-ambient-update.adoc

```diff
@@ -0,0 +1,784 @@
+NAME      REVISIONS   READY   IN USE   ACTIVE REVISION   STATUS    VERSION   AGE
+default   1           1       1        default           Healthy   v1.26.3   7d
+----
+
+*Procedure:*
+
+. Update the Istio resource version. For example, to update to Istio 1.26.4, set the `spec.version` field to `v1.26.4` by running the following command:
++
+[source,bash]
+----
+$ kubectl patch istio default -n istio-system --type='merge' -p '{"spec":{"version":"v1.26.4"}}'
+----
+
+. Monitor the istiod pod replacement. The old pod will be terminated and a new pod will be created:
++
+[source,bash]
+----
+$ kubectl get pods -n istio-system -l app=istiod -w
+----
+
+. Wait for the Istio control plane to become ready:
++
+[source,bash]
+----
+$ kubectl wait --for=condition=Ready istios/default -n istio-system --timeout=5m
+----
+
+. Verify the Istio resource shows the new version and is in a Healthy state:
++
+[source,bash]
+----
+$ kubectl get istio -n istio-system
+NAME      REVISIONS   READY   IN USE   ACTIVE REVISION   STATUS    VERSION   AGE
+default   1           1       1        default           Healthy   v1.26.4   7d1h
+----
+
+. Confirm the istiod pod is running with the new version:
++
+[source,bash]
+----
+$ kubectl get pods -n istio-system -l app=istiod
+NAME                             READY   STATUS    RESTARTS   AGE
+istiod-default-6bd6b8664b-x7k2m  1/1     Running   0          2m15s
+----
+
+. Verify the control plane is functioning correctly by checking the istiod logs:
++
+[source,bash]
+----
+$ kubectl logs -n istio-system -l app=istiod --tail=50 | grep -i "version\|ready"
+----
+
+After updating the Istio control plane, proceed to update IstioCNI, ZTunnel, verify ambient workloads, and optionally update Waypoint Proxies. Refer to section "Common Update Procedures for Ambient Components" for detailed instructions on updating these components.
+
+[[recommendations-for-inplace-strategy-in-ambient-mode]]
+=== Recommendations for InPlace strategy in Ambient Mode
+
+* **High Availability:** Configure the istiod deployment with multiple replicas for high availability during updates. See the link:../general/istiod-ha.adoc#running-istiod-in-ha-mode[Istiod HA guide] for more information.
+* **ZTunnel Updates:** The ZTunnel DaemonSet uses a RollingUpdate strategy by default, which updates pods one node at a time. Monitor the rollout to ensure it completes successfully.
+* **Maintenance Window:** While ambient mode is designed to minimize disruption, it's recommended to perform upgrades during a maintenance window.
+* **Testing:** Always test the upgrade process in a non-production environment first.
+
+'''
+
+[[about-revisionbased-strategy]]
+== About RevisionBased strategy
+
+The RevisionBased strategy runs two revisions of the control plane during an upgrade. This approach supports gradual workload migration from the old control plane to the new one, enabling canary upgrades. It also supports upgrades across more than one minor version.
```

Comment:
> It also supports upgrades across more than one minor version.

This is not guaranteed to work in Ambient as "The ztunnel at version 1.x is compatible with the control plane at version 1.x+1 and 1.x. This means the control plane must be upgraded before ztunnel, as long as their version difference is within one minor version. "

I feel we can remove the RevisionBased upgrades for now.
CC: @dgn @fjglira